core/src/test/java/org/apache/iceberg/avro/TestAvroEnums.java

```diff
@@ -71,8 +72,8 @@ public void writeAndValidateEnums() throws IOException {
     }
 
     Schema schema = new Schema(AvroSchemaUtil.convert(avroSchema).asStructType().fields());
-    List<GenericData.Record> rows;
-    try (AvroIterable<GenericData.Record> reader =
+    List<Record> rows;
+    try (AvroIterable<Record> reader =
```

Comment:
nit: I think it would generally be good to not apply changes that aren't related to Junit5 migration